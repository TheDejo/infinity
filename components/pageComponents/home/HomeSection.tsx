import React from 'react';
import JobCard from '../../common/blocks/JobCard';
import Select from '../../common/components/Select';
import Hero from './Hero';

const HomeSection: React.FC = () => {
    const BUTTONS = [
        {
            title: 'Company',
            key: 'company' as const,
            options: [],
        },
        {
            title: 'Title',
            key: 'title' as const,
            options: [],
        },
        {
            title: 'Skills',
            key: 'company' as const,
            options: [],
        },
        {
            title: 'Job Type',
            key: 'jobType' as const,
            options: [],
        },
        {
            title: 'Location',
            key: 'location' as const,
            options: [
                {
                    name: 'Yes',
                    value: 'Yes',
                },
                {
                    name: 'No',
                    value: 'No',
                },
            ],
        },
    ];

    return (
        <div>
            <Hero />
            <section className="p-8">
                <h1 className="font-medium">Filter</h1>
                <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-5 gap-4 py-6">
                    {BUTTONS.map(({ title, key, options }) => (
                        <Select key={key} title={title} data={options} />
                    ))}
                </div>
                <div className="border border-gray-200 my-4" />
                <JobCard />
            </section>
        </div>
    );
};

export default HomeSection;
