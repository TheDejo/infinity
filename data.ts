const JOBS_DATA = [
    {
        id: '1',
        title: 'Frontend Developer',
        company:'Vaarst',
        description: 'We are looking for a skilled Frontend Developer to be a part our growing web platform team. You will be facing interesting problems that require skill and creativity. As a team we are passionate about our company’s future and work hard to see the results of our efforts changing our industry. We are small enough that everybody makes a difference but big enough to provide variety of work and strong in-house expertise. We work together and celebrate our successes together.',
        skills: ['TypeScript', 'React', 'Three.js'],
        type: ['Full Time'],
        location: 'Bristol BS1',
    },
    {
        id: '2',
        title: 'Frontend Developer',
        company:'Develop',
        description: 'We are recruiting on behalf of our client who are a global digital agency, who are looking to hire multiple React Developers on a contract basis for various long term projects. They need someone who is a strong react developer but also has understanding of PHP so that you can fix APIs',
        skills: ['React', 'Typescript', 'Gatsby'],
        type: ['Remote', 'Parmanent'],
        location: 'Bristol',
    },
    {
        id: '3',
        title: 'Frontend Developer',
        company:'Dotmatics',
        description: 'Collaborate with product management, application science and support stakeholders to develop and maintain product suite. Deliver high quality clean code and technical documentation. Solve challenging customer issues using a rapid delivery model. Develop prototype/proof-of-concept implementations for early feedback.Pursue continual improvement in all aspects of the software and its development lifecycle',
        skills: ['JavaScript', 'React'],
        type: ['Remote'],
        location: 'Weschester',
    },
    {
        id: '4',
        title: 'Backend Developer',
        company:'Clowdr CIC',
        description: 'Our ideal candidate for this position is a is an experienced developer with a particular interest in back-end design and implementation. You should have an eye for what looks good and an instinct for good user experience.',
        skills: ['Typescript', 'React', 'GraphQL'],
        type: ['Full Time'],
        location: 'Liverpool',
    },
    {
        id: '5',
        title: 'Software Engineer',
        company:'Econscia',
        description: 'You will be working on different challenges day to day. This could be getting deep into research from which you’ll develop prototypes and POCs, working on key features alongside the team, or contributing to a clean, enterprise-grade SaaS application with security by design. We hope you can work with us to own key aspects of econscia’s analytics engine, but could also roll up your sleeves and dive into research, onboard customers, or dabble in unfamiliar technologies quickly - when the job needs to get done. Over time, the opportunity is yours to shape this role as we grow.',
        skills: ['Django', 'Javascript'],
        type: ['Remote', 'Contract'],
        location: 'London',
    },
    {
        id: '6',
        title: 'Junior Backend Developer',
        company:'Gear4Music',
        description: 'As a JavaScript junior developer, you will be responsible for Google Ads API developments and the implementation and maintenance of scripts, as well as reporting automation. These solutions will work to automate targeting, data sourcing and optimisation across multiple Google Ads accounts. Working closely with our marketing executives, you will help other SEM team members design scripts, automate & optimise their PPC accounts.',
        skills: ['Javascript'],
        type: ['Remote'],
        location: 'York YO26',
    },
    {
        id: '7',
        title: 'Production Manager',
        company:'BBC',
        description: 'Panorama is a ground breaking current affairs investigative series and we are looking for an experienced Production Manager with experience of managing a large portfolio or strand to join the team. Panorama produces some hard hitting investigations from undercover reporting to high risk filming in hostile countries covering a wide range content – experience of similar would be an advantage. You will work closely with third parties and Indies that are commissioned. Panorama produces 39 half hours and 7 one hours per year produced either inhouse London, Belfast, Regions or by Indies. This is an exciting opportunity for the right person to join the team.',
        skills: ['Management'],
        type: ['Contract'],
        location: 'London W1A',
    },
    {
        id: '8',
        title: 'Software Engineer',
        company:'Alliants',
        description: 'This position will see you working with an experienced team of developers, to help global brands improve their customer experiences through technology.',
        skills: ['Unit tests', 'Git', 'Node Js'],
        type: ['Remote'],
        location: 'Southampton',
    },
    {
        id: '9',
        title: 'Backend Developer',
        company:'KOBAS',
        description: 'The Kobas team provides Software-as-a-Service to the hospitality industry, transforming operational routines from dull, spreadsheet-based grind to quick, point-and-click management, across our core product pillars: In-Venue, Operations, Customers, Teams and Insights. Our software helps businesses process millions of pounds of sales each year. We unite with our clients so that they can benefit from our industry-leading technology, as well as ensuring that best practices can be followed, sharing knowledge amongst our clients, providing insight, and achieving maximum ROI.',
        skills: ['AWS', 'PHP'],
        type: ['Remote', 'Contract'],
        location: 'Home Based',
    },
    {
        id: '10',
        title: 'Software Developer',
        company:'Sainsburys',
        description: 'As a Software Engineer, you will build and maintain a suite of high performant and robust APIs that provide an outstanding user experience for our customers. You will work as part of an agile development team, helping to build a high performing culture where quality matters and delivery happens at pace.',
        skills: ['NodeJS', 'Typescript'],
        type: ['Full Time'],
        location: 'Edinburgh',
    },
]

// eslint-disable-next-line import/no-anonymous-default-export
export default {
    JOBS_DATA,
}