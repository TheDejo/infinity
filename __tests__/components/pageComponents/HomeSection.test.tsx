import React from 'react';
import { render, Matcher, MatcherOptions, screen, act, fireEvent } from '@testing-library/react';
import HomeSection from '@/pageComponents/home/HomeSection';
import renderer from 'react-test-renderer';
import data from '@data';

const mockedPush = jest.fn();

jest.mock('next/router', () => ({
    useRouter() {
        return {
            pathname: '/1',
            push: mockedPush,
        };
    },
}));

describe('Home Section  Component', () => {
    let getByTestId: (
        text: Matcher,
        options?: MatcherOptions | undefined,
        waitForElementOptions?: unknown,
    ) => HTMLElement;

    beforeEach(() => {
        const component = render(<HomeSection />);
        getByTestId = component.getByTestId;
    });

    it('should close the modal when the backdrop is clicked', () => {
        const applyBtn = screen.getAllByText('Apply Now')[0];
        expect(applyBtn).toBeInTheDocument();

        act(() => {
            fireEvent.click(applyBtn);
        });
        const modal = getByTestId('main-modal');
        expect(modal).toBeInTheDocument();

        const underlay = getByTestId('modal-underlay');
        expect(underlay).toBeInTheDocument();

        act(() => {
            fireEvent.mouseDown(underlay);
        });
        expect(modal).not.toBeInTheDocument();

        expect(underlay).not.toBeInTheDocument();
    });

    it('filters data when the search input is clicked', () => {
        const searchJob = getByTestId('hero-input') as HTMLInputElement;

        act(() => {
            fireEvent.change(searchJob, { target: { value: 'frontend Developer' } });
        });
        expect(searchJob.value).toBe('frontend Developer');

        const searchButton = screen.getByText('Find Jobs');

        act(() => {
            fireEvent.click(searchButton);
        });

        expect(screen.getByTestId('Frontend Developer')).toBeInTheDocument();
    });

    it('pushes user to dynamic path when clicked', () => {
        const component = render(<HomeSection />);
        const button = screen.getAllByText('View Job')[0];
        act(() => {
            fireEvent.click(button);
        });

        expect(mockedPush).toBeCalledTimes(1);
        expect(mockedPush).toBeCalledWith('1');
    });

    it('matches Single input snapshot', () => {
        const tree = renderer.create(<HomeSection />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
