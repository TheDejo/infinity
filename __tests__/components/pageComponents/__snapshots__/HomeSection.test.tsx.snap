// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Section  Component matches Single input snapshot 1`] = `
<section
  data-testid="home-section"
>
  <section
    className="lg:px-8"
    test-dataId="hero-section"
  >
    <div
      className="p-8 bg-infinity-hero lg:rounded-[30px] flex justify-center items-center min-h-[30vh]"
    >
      <div
        className="mr-6"
      >
        <h1
          className="xl:w-[30rem] font-semibold text-2xl md:text-3xl lg:text-6xl mb-3"
        >
          Welcome To Infinity, & Beyond
        </h1>
        <p
          className="md:w-[30rem] font-normal text-sm md:text-base lg:text-xl"
        >
          Connect Today With Innovative Companies That Care About Planetary Diversity & Inclusion
        </p>
        <form
          className="flex flex-col md:flex-row mt-6 justify-start w-full"
        >
          <input
            className="rounded-lg  w-full md:w-72 xl:w-96 pl-6 py-2  placeholder-gray-600"
            data-testid="hero-input"
            name="search"
            onChange={[Function]}
            placeholder="Job Title or keywords"
            type="text"
            value=""
          />
          <button
            className="hidden md:flex justify-center items-center text-white text-lg rounded-lg bg-[#F3A494] px-5  shadow-infinity ml-2"
            onClick={[Function]}
          >
            Find Jobs
          </button>
        </form>
      </div>
      <figure
        className="hidden md:flex"
      >
        <div
          style={
            Object {
              "boxSizing": "border-box",
              "display": "inline-block",
              "margin": 0,
              "maxWidth": "100%",
              "overflow": "hidden",
              "position": "relative",
            }
          }
        >
          <div
            style={
              Object {
                "boxSizing": "border-box",
                "display": "block",
                "maxWidth": "100%",
              }
            }
          >
            <img
              alt=""
              aria-hidden={true}
              src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNDc4IiBoZWlnaHQ9IjQzOCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
              style={
                Object {
                  "border": "none",
                  "display": "block",
                  "margin": 0,
                  "maxWidth": "100%",
                  "padding": 0,
                }
              }
            />
          </div>
          <img
            alt="hero image"
            data-nimg="intrinsic"
            decoding="async"
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style={
              Object {
                "border": "none",
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": 0,
                "left": 0,
                "margin": "auto",
                "maxHeight": "100%",
                "maxWidth": "100%",
                "minHeight": "100%",
                "minWidth": "100%",
                "objectFit": undefined,
                "objectPosition": undefined,
                "padding": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": 0,
              }
            }
          />
          <noscript>
            <img
              alt="hero image"
              data-nimg="intrinsic"
              decoding="async"
              loading="lazy"
              src="/_next/image?url=%2Fassets%2Fimages%2Fhero.png&w=1080&q=75"
              srcSet="/_next/image?url=%2Fassets%2Fimages%2Fhero.png&w=640&q=75 1x, /_next/image?url=%2Fassets%2Fimages%2Fhero.png&w=1080&q=75 2x"
              style={
                Object {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": undefined,
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
          </noscript>
        </div>
      </figure>
    </div>
  </section>
  <div
    className="py-8"
  >
    <div
      className="px-8"
    >
      <h1
        className="font-medium"
      >
        Filter
      </h1>
      <div
        className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-5 gap-4 py-6"
      >
        <div
          className="bg-black text-sm py-3 px-4 flex justify-center items-center rounded-lg shadow-infinity outline-none border-none"
        >
          <select
            className="bg-black text-white w-full after:border-none"
            onChange={[Function]}
          >
            <option
              disabled={true}
              selected={true}
              value=""
            >
              Company
            </option>
            <option
              value="Microsoft"
            >
              Microsoft
            </option>
            <option
              value="Facebook"
            >
              Facebook
            </option>
            <option
              value="Twitter"
            >
              Twitter
            </option>
            <option
              value="Google"
            >
              Google
            </option>
            <option
              value="Space X"
            >
              Space X
            </option>
          </select>
        </div>
        <div
          className="bg-black text-sm py-3 px-4 flex justify-center items-center rounded-lg shadow-infinity outline-none border-none"
        >
          <select
            className="bg-black text-white w-full after:border-none"
            onChange={[Function]}
          >
            <option
              disabled={true}
              selected={true}
              value=""
            >
              Title
            </option>
            <option
              value="Frontend Developer"
            >
              Frontend Developer
            </option>
            <option
              value="Backend Developer"
            >
              Backend Developer
            </option>
            <option
              value="Product Manager"
            >
              Product Manager
            </option>
            <option
              value="Software Developer"
            >
              Software Developer
            </option>
          </select>
        </div>
        <div
          className="bg-black text-sm py-3 px-4 flex justify-center items-center rounded-lg shadow-infinity outline-none border-none"
        >
          <select
            className="bg-black text-white w-full after:border-none"
            onChange={[Function]}
          >
            <option
              disabled={true}
              selected={true}
              value=""
            >
              Skills
            </option>
            <option
              value="React"
            >
              React
            </option>
            <option
              value="TypeScript"
            >
              TypeScript
            </option>
            <option
              value="Node Js"
            >
              Node Js
            </option>
            <option
              value="Express"
            >
              Express
            </option>
            <option
              value="Node"
            >
              Node
            </option>
          </select>
        </div>
        <div
          className="bg-black text-sm py-3 px-4 flex justify-center items-center rounded-lg shadow-infinity outline-none border-none"
        >
          <select
            className="bg-black text-white w-full after:border-none"
            onChange={[Function]}
          >
            <option
              disabled={true}
              selected={true}
              value=""
            >
              Job Type
            </option>
            <option
              value="Fully Remote"
            >
              Fully Remote
            </option>
            <option
              value="Full time"
            >
              Full time
            </option>
            <option
              value="Contract"
            >
              Contract
            </option>
            <option
              value="Remote"
            >
              Remote
            </option>
          </select>
        </div>
        <div
          className="bg-black text-sm py-3 px-4 flex justify-center items-center rounded-lg shadow-infinity outline-none border-none"
        >
          <select
            className="bg-black text-white w-full after:border-none"
            onChange={[Function]}
          >
            <option
              disabled={true}
              selected={true}
              value=""
            >
              Location
            </option>
            <option
              value="4, Adeniyan Ogunsanya, okokomaiko"
            >
              4, Adeniyan Ogunsanya, okokomaiko
            </option>
            <option
              value="6, Bode Thomas, Close"
            >
              6, Bode Thomas, Close
            </option>
            <option
              value="7, Ikorodu close"
            >
              7, Ikorodu close
            </option>
            <option
              value="Lagos"
            >
              Lagos
            </option>
          </select>
        </div>
      </div>
    </div>
    <div
      className="border border-gray-200 my-4"
    />
    <section
      className="mx-8"
    >
      <div
        className="p-8 w-full min-h-[330px] border border-black rounded-3xl flex flex-col mb-8"
      >
        <div
          className="flex text-left w-full"
        >
          <h2
            className="font-medium text-2xl"
            data-testid="Frontend Developer"
          >
            Frontend Developer
          </h2>
        </div>
        <div
          className="border border-gray-200 my-4"
        />
        <div
          className="grid grid-cols-1 md:grid-cols-2 gap-5"
        >
          <div
            className="flex md:flex-row flex-col"
          >
            <p
              className="md:mr-3 mb-3 md:mb-0 w-32 text-darkBlue font-semibold"
            >
              Company
              :
            </p>
            <p>
              Microsoft
            </p>
          </div>
          <div
            className="flex md:flex-row flex-col"
          >
            <p
              className="md:mr-3 mb-3 md:mb-0 w-32 text-darkBlue font-semibold"
            >
              Description
              :
            </p>
            <p>
              nulla pellentesque dignissim enim sit amet venenatis urna cursus eget nunc scelerisque viverra mauris in aliquam sem fringilla ut morbi
            </p>
          </div>
          <div
            className="flex md:flex-row flex-col"
          >
            <p
              className="md:mr-3 mb-3 md:mb-0 w-32 text-darkBlue font-semibold"
            >
              Job Type
              :
            </p>
            <p>
              Fully Remote
            </p>
          </div>
          <div
            className="flex md:flex-row flex-col"
          >
            <p
              className="md:mr-3 mb-3 md:mb-0 w-32 text-darkBlue font-semibold"
            >
              Skills
              :
            </p>
            <p>
              React
              TypeScript
            </p>
          </div>
          <div
            className="flex md:flex-row flex-col"
          >
            <p
              className="md:mr-3 mb-3 md:mb-0 w-32 text-darkBlue font-semibold"
            >
              Location
              :
            </p>
            <p>
              4, Adeniyan Ogunsanya, okokomaiko
            </p>
          </div>
        </div>
        <div
          className="border border-gray-200 my-4"
        />
        <div
          className="w-full flex md:flex-row flex-col justify-end"
        >
          <button
            className="px-6 py-2 mb-3 md:mb-0 mr-2 rounded-3xl hover:scale-110	flex justify-center items-center text-white bg-black"
            onClick={[Function]}
          >
            Apply Now
          </button>
          <button
            className="px-6 py-2 mb-3 md:mb-0 mr-2 rounded-3xl hover:scale-110	flex justify-center items-center text-black bg-[#FCF0EC]"
            onClick={[Function]}
          >
            View Job
          </button>
        </div>
      </div>
    </section>
    <section
      className="mx-8"
    >
      <div
        className="p-8 w-full min-h-[330px] border border-black rounded-3xl flex flex-col mb-8"
      >
        <div
          className="flex text-left w-full"
        >
          <h2
            className="font-medium text-2xl"
            data-testid="Backend Developer"
          >
            Backend Developer
          </h2>
        </div>
        <div
          className="border border-gray-200 my-4"
        />
        <div
          className="grid grid-cols-1 md:grid-cols-2 gap-5"
        >
          <div
            className="flex md:flex-row flex-col"
          >
            <p
              className="md:mr-3 mb-3 md:mb-0 w-32 text-darkBlue font-semibold"
            >
              Company
              :
            </p>
            <p>
              Facebook
            </p>
          </div>
          <div
            className="flex md:flex-row flex-col"
          >
            <p
              className="md:mr-3 mb-3 md:mb-0 w-32 text-darkBlue font-semibold"
            >
              Description
              :
            </p>
            <p>
              nulla pellentesque dignissim enim sit amet venenatis urna cursus eget nunc scelerisque viverra mauris in aliquam sem fringilla ut morbi
            </p>
          </div>
          <div
            className="flex md:flex-row flex-col"
          >
            <p
              className="md:mr-3 mb-3 md:mb-0 w-32 text-darkBlue font-semibold"
            >
              Job Type
              :
            </p>
            <p>
              Full time
            </p>
          </div>
          <div
            className="flex md:flex-row flex-col"
          >
            <p
              className="md:mr-3 mb-3 md:mb-0 w-32 text-darkBlue font-semibold"
            >
              Skills
              :
            </p>
            <p>
              Node Js
              Express
            </p>
          </div>
          <div
            className="flex md:flex-row flex-col"
          >
            <p
              className="md:mr-3 mb-3 md:mb-0 w-32 text-darkBlue font-semibold"
            >
              Location
              :
            </p>
            <p>
              6, Bode Thomas, Close
            </p>
          </div>
        </div>
        <div
          className="border border-gray-200 my-4"
        />
        <div
          className="w-full flex md:flex-row flex-col justify-end"
        >
          <button
            className="px-6 py-2 mb-3 md:mb-0 mr-2 rounded-3xl hover:scale-110	flex justify-center items-center text-white bg-black"
            onClick={[Function]}
          >
            Apply Now
          </button>
          <button
            className="px-6 py-2 mb-3 md:mb-0 mr-2 rounded-3xl hover:scale-110	flex justify-center items-center text-black bg-[#FCF0EC]"
            onClick={[Function]}
          >
            View Job
          </button>
        </div>
      </div>
    </section>
    <section
      className="mx-8"
    >
      <div
        className="p-8 w-full min-h-[330px] border border-black rounded-3xl flex flex-col mb-8"
      >
        <div
          className="flex text-left w-full"
        >
          <h2
            className="font-medium text-2xl"
            data-testid="Product Manager"
          >
            Product Manager
          </h2>
        </div>
        <div
          className="border border-gray-200 my-4"
        />
        <div
          className="grid grid-cols-1 md:grid-cols-2 gap-5"
        >
          <div
            className="flex md:flex-row flex-col"
          >
            <p
              className="md:mr-3 mb-3 md:mb-0 w-32 text-darkBlue font-semibold"
            >
              Company
              :
            </p>
            <p>
              Twitter
            </p>
          </div>
          <div
            className="flex md:flex-row flex-col"
          >
            <p
              className="md:mr-3 mb-3 md:mb-0 w-32 text-darkBlue font-semibold"
            >
              Description
              :
            </p>
            <p>
              nulla pellentesque dignissim enim sit amet venenatis urna cursus eget nunc scelerisque viverra mauris in aliquam sem fringilla ut morbi
            </p>
          </div>
          <div
            className="flex md:flex-row flex-col"
          >
            <p
              className="md:mr-3 mb-3 md:mb-0 w-32 text-darkBlue font-semibold"
            >
              Job Type
              :
            </p>
            <p>
              Contract
            </p>
          </div>
          <div
            className="flex md:flex-row flex-col"
          >
            <p
              className="md:mr-3 mb-3 md:mb-0 w-32 text-darkBlue font-semibold"
            >
              Skills
              :
            </p>
            <p>
              React
              TypeScript
            </p>
          </div>
          <div
            className="flex md:flex-row flex-col"
          >
            <p
              className="md:mr-3 mb-3 md:mb-0 w-32 text-darkBlue font-semibold"
            >
              Location
              :
            </p>
            <p>
              7, Ikorodu close
            </p>
          </div>
        </div>
        <div
          className="border border-gray-200 my-4"
        />
        <div
          className="w-full flex md:flex-row flex-col justify-end"
        >
          <button
            className="px-6 py-2 mb-3 md:mb-0 mr-2 rounded-3xl hover:scale-110	flex justify-center items-center text-white bg-black"
            onClick={[Function]}
          >
            Apply Now
          </button>
          <button
            className="px-6 py-2 mb-3 md:mb-0 mr-2 rounded-3xl hover:scale-110	flex justify-center items-center text-black bg-[#FCF0EC]"
            onClick={[Function]}
          >
            View Job
          </button>
        </div>
      </div>
    </section>
    <div
      className="w-full flex justify-center items-center"
    >
      <button
        className="w-[40px] h-[40px] border border-[#c4c4c4] font-semibold rounded-md mr-2 hover:scale-110"
        onClick={[Function]}
      >
        1
      </button>
      <button
        className="w-[40px] h-[40px] border border-[#c4c4c4] font-semibold rounded-md mr-2 hover:scale-110"
        onClick={[Function]}
      >
        2
      </button>
    </div>
  </div>
</section>
`;
